app/components/links/TextLink.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link";
+import React from "react";
+
+export const TextLink = ({ url, text }: { url: string; text: string }) => {
+  return (
+    <Link
+      href={url}
+      className="font-poppins font-normal text-[14.39px] leading-[21.59px] text-buttonColor"
+    >
+      {text}
+    </Link>
+  );
+};
```

Comment:
export defaults separately
```suggestion
  );
};

export default TextLink;
```

Reply:
done